templates/about.html

```diff
@@ -27,13 +27,13 @@ <h3 class="push-1-top">PSSI and PSF (<a target="_blank" href="https://www.python
         <p>It is also worth mentioning that PSSI has received funds from PSF for conducting PyCon India in the past and might receive such funds in the future as well. However PSF has no say in the governance and conduct of the board members of PSSI.</p>
 
         <h3 class="push-1-top">PSSI Board</h3>
-        <p>The board positions of PSSI </p>
+        <p>The board positions of PSSI to which annual elections are held every year include,</p>
```

Comment:
As per new bye law election will be held every two years. I think for now can you omit this sentence ?